Demo/build.gradle.kts

```diff
@@ -14,8 +14,12 @@ kotlin {
         implementation(cup.sourceCode)
         implementation(cup.plugin.laser)
         implementation(cup.plugin.speakerWindow)
+        implementation(cup.widget.material)
+        implementation(cup.widget.material3)
```

Comment:
The demo should use either `material` or `material3` but not both.
`material` is probably better as a large part of `material3` is still experimental in Compose Multiplatform.

If we want to test `material3`, we should create a `demo-material3` project.